src/apps/search-header/search-header.dev.tsx

```diff
@@ -7,9 +7,9 @@ export default {
   title: "Apps / Search Header",
   component: SearchHeaderEntry,
   argTypes: {
-    searchHeaderUrlText: {
+    baseUrl: {
       name: "Search header base URL",
-      defaultValue: "https://bibliotek.dk/search",
+      defaultValue: "https://bibliotek.dk",
```

Comment:
Consider introducing separate props from the material page and the search page.

The current approach integrates logic of routing structures of the mounting app into the React components. We would like to avoid this. One way to do this is to have separate props for search results and materials.

Another aspect of this would be placeholder support for querys, ids etc. so we can define where to put query, (work) id etc. in the prop value. [In the current React components this is implemented with `:placeholder`](https://github.com/danskernesdigitalebibliotek/ddb-react/blob/master/src/core/replacePlaceholders.js). This would make it possible to have the search prop be something `/search?q=:query` in the CMS but something like `https://danskernesdigitalebibliotek.github.io/dpl-react/iframe.html?args=query::query&id=apps-search-result--search-result&viewMode=story` in Storybook.